tests/soak_test/start_server.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+HAZELCAST_TEST_VERSION="4.0"
+HAZELCAST_VERSION="4.0"
+PID=$$
+
+mkdir -p "server_logs"
+
+CLASSPATH="hazelcast-${HAZELCAST_VERSION}.jar:hazelcast-${HAZELCAST_TEST_VERSION}-tests.jar"
+CMD_CONFIGS="-Dhazelcast.multicast.group=224.206.1.1 -Djava.net.preferIPv4Stack=true"
```

Comment:
nit: we already define multicast group in `hazelcast.xml`, so probably `-Dhazelcast.multicast.group=224.206.1.1` could be omitted.